kernel/memory_x86_64/src/lib.rs

```diff
@@ -182,14 +176,15 @@ pub fn find_section_memory_bounds(boot_info: &impl BootInformation) -> Result<(A
                 data_flags = Some(flags);
                 "nano_core .bss"
             }
-            ".page_table" => {
-                // We bootstrap the page table from the CR3 register.
+            // This appears when compiling for BIOS.
+            ".page_table" | ".stack" => {
                 continue;
```

Comment:
for the sake of completeness to 
```suggestion
                debug!("     no need to map this section, it is mapped separately later");
                continue;
```